recipe/meta.yaml

```diff
@@ -41,8 +41,8 @@ requirements:
     - zlib
     - curl
     - pcre
-    - gshhg-gmt
-    - dcw-gmt
+    - gshhg-gmt  # [not win]
+    - dcw-gmt  # [not win]
```

Comment:
:+1: 

Do we know why the Linux/macOS strategy did not work on Windows?

Reply:
Not really. I have close to zero Windows experience with these things and don't even know where to start looking.